eox_tenant/api/v1/serializers.py

```diff
@@ -0,0 +1,50 @@
+"""
+Serializers file for eox_tenant api v1.
+"""
+from rest_framework import serializers
+
+from eox_tenant.models import Microsite, Route, TenantConfig
+
+
+class MicrositeSerializer(serializers.ModelSerializer):
+    """
+    Serializer class for Microsite model.
+    """
+    values = serializers.JSONField()
+
+    class Meta:
+        """
+        Define the serializer fields.
+        """
+        model = Microsite
+        fields = '__all__'
+
+
+class TenantConfigSerializer(serializers.ModelSerializer):
+    """
+    Serializer class for TenantConfig model.
+    """
+    lms_configs = serializers.JSONField()
+    studio_configs = serializers.JSONField()
+    theming_configs = serializers.JSONField()
+    meta = serializers.JSONField()
+
+    class Meta:
+        """
+        Define the serializer fields.
+        """
+        model = TenantConfig
+        fields = '__all__'
+
+
+class RouteSerializer(serializers.ModelSerializer):
+    """
+    Serializer class for Route model.
+    """
+
```

Comment:
Is this emtpy on purpose?

Reply:
yes

Reply:
Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line. https://www.python.org/dev/peps/pep-0257/
indeed it's missing in in the other classes of this file